dolo/compiler/model_numeric.py

```diff
@@ -179,6 +179,14 @@ def _repr_html_(self):
             equations = {"dynare": self.symbolic.equations}
         else:
             equations = self.symbolic.equations
+            # Create definitions equations and append to equations dictionary
+            definitions = self.symbolic.definitions
+            tmp = []
+            for deftype in definitions:
+                tmp.append(deftype + '=' + definitions[deftype])
+            definitions = {'definitions': tmp}
+            equations.update(definitions)
```

Comment:
Ya, all local. I mean, we can split them up to make it clearer, but it would be at the cost of additional if statements later on in the function.